File: 2/main.c
#include <stdio.h>
// Euler project fibonacci even sum with 4M max.
int main(void){
  int cap = 4000000;
  int even_sum = 0;
  int i = 1;
  int prev = 0;
  int next;
  
  while (i < cap) {
    next = prev + i;
    prev=i;
    i=next;
    
    if (i % 2 == 0) {
      even_sum += i;
    }

    printf("%i\n %i", i, even_sum);
  }
  

  printf("Hello World\n");

  return 0;
}

File: 2/main.py
# problem_2.py
a, b = 1, 2
sum = 0
while a <= 4000000:
    if a % 2 == 0:
        sum += a
    a, b = b, a + b

print(f"The sum of even terms in the Fibonacci sequence that do not exceed four million is: {sum}")
